uwblueprint/project-read-frontend
# Set up shell for the Create Session page

### Notion ticket link



[Set up create session routes and stepper](https://www.notion.so/uwblueprintexecs/Developer-Hub-30e9b6c82af24054b31acf7c5e822c89?p=2f49b7e5cb30468fb6f439f4c63b9506)



### Implementation description

- creates a `CreateSession` component that will be the 🏠 for the 3 steps to create a session
- registered a route for it (/sessions/create) that's linked when users click "Add new session" in the session selector
- added tests ✨ 



### Steps to test

1. open up the deploy preview
2. navigate to the sessions page and click "Add new session"
3. play around! 🤸‍♀️ 
    1. the stepper should update as you click back/next
    2. the back button is disabled on the first step
    3. clicking "Finish" on the third step takes you back to the sessions page



### What should reviewers focus on?

- code readability & cleanliness

### Checklist
